tgui/packages/tgui-panel/ping/PingIndicator.js

```diff
@@ -19,9 +20,16 @@ export const PingIndicator = (props) => {
   ]);
   const roundtrip = ping.roundtrip ? toFixed(ping.roundtrip) : '--';
   return (
-    <div className="Ping">
+    <Button
+      lineHeight="15px"
+      width="50px"
+      className="Ping"
+      color="transparent"
+      tooltip="Ping relays"
+      tooltipPosition="bottom-start"
+      onClick={() => act('ping_relays')}>
       <Box className="Ping__indicator" backgroundColor={color} />
```

Comment:
The ping indicator circle seems to have been offset down a little bit:
**Before (master)/After (this PR):**
![before pre-pr](https://github.com/cmss13-devs/cmss13/assets/57483089/54ca008c-5091-4600-93e6-04780c8bc1a2) ![before](https://github.com/cmss13-devs/cmss13/assets/57483089/6d95697c-c19f-439d-8851-b28ab0e1b5d2)
<hr>

Tweaking `Ping__indicator`'s `top` value in `tgui\packages\tgui-panel\styles\components\Ping.scss` seems to work to get it back in the centre:
```diff
.Ping__indicator {
  content: '';
  position: absolute;
-  top: 0.5em;
+  top: 0.4em;
```
**Before/After:**
![before](https://github.com/cmss13-devs/cmss13/assets/57483089/3a83d937-e31a-4cc8-b0ea-695972acafb3) ![after](https://github.com/cmss13-devs/cmss13/assets/57483089/ad653de2-b451-4f75-a476-104aa5b3334c)

Reply:
light theme changes the padding on the button; so I will just force the padding to how dark theme does it here.